moon/cfc_chat_transit/server/modules/startup.moon

```diff
@@ -0,0 +1,18 @@
+import guard from ChatTransit
+import GetMap from game
+
+ChatTransit.Initialize = (data) =>

```

Comment:
`Initialize` has some code connotations, I think it'd be better to name it a little more bluntly
```suggestion
ChatTransit.MapStartup = (data) =>
```

Reply:
Along with `hook.Add "Initialize", "CFC_ChatTransit_StartupListener", guard ChatTransit\MapStartup` of course